ajax/libs/zingchart-angularjs/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "zingchart-angularjs",
+  "filename": "zingchart-angularjs.min.js",
+  "version": "1.2.0",
+  "description": "A ZingChart AngularJS directive",
+  "keywords": [
+    "zingchart-angularjs"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/zingchart/ZingChart-AngularJS"
+  },
+  "author": "ZingChart",
+  "license": "MIT",
+  "npmName": "zingchart-angularjs",
+  "npmFileMap": [
+    {
+      "basePath": "src",
+      "files": [
+        "**/*"
```

Comment:
@NiniHsu I think it's better to use `zingchart-angularjs*` here, `**/*` basically is for `dist/` or `build/` directory.